ui/src/learningReflections/templates/learningReflections-storyEditor.handlebars

```diff
@@ -44,10 +44,10 @@
                     </div>
                 </div>
                 <div class="sjrk-st-button-container">
-                    <a href="javascript:void(0)" class="sjrk-st-story-editor-next sjrkc-st-story-editor-next">
+                    <button type="button" class="sjrk-st-story-editor-next sjrkc-st-story-editor-next" aria-role="{{localizedMessages.message_continue}}">
```

Comment:
It seems making more sense to use the keyword `aria-label` here for indicating the button usage rather than `aria-role`.

The same comment applies to all uses of `aria-role` below.